### DISCUSSION TITLE
figures / images

### DISCUSSION THREAD

**Discussion by raserdk** [Berny23/LD-ToyPad-Emulator#111] [Ideas]
insted of text, would it be possible to have images of the figures, and charecters on the dropdown and on the the pad instead, so my son could do this himself

**Comment by Berny23** [comment]
I will not add copyrighted files to this repository, too many fan projects have been taken down.

**Reply by raserdk** [reply]
oh i see then, 
how would i do it myself for personal use, if you want to tell me, if not its ok, and i understand why you wont do it
